src/core/utils/building/resolve-config.ts

```diff
@@ -241,24 +241,15 @@ function mergeInlineConfig(
     srcDir: inlineConfig.srcDir ?? userConfig.srcDir,
     outDir: inlineConfig.outDir ?? userConfig.outDir,
     zip,
-    analysis: {
-      ...userConfig.analysis,
-      ...inlineConfig.analysis,
-    },
-    alias: {
-      ...userConfig.alias,
-      ...inlineConfig.alias,
-    },
-    experimental: {
-      ...userConfig.experimental,
-      ...inlineConfig.experimental,
-    },
+    analysis: defu(inlineConfig.analysis ?? {}, userConfig.analysis ?? {}),
```

Comment:
Hmm, other than some one-off values, everything here is defu-able... Instead of doing individual defus, I could probably do a single one?

Reply:
Released in [v0.17.9](https://github.com/wxt-dev/wxt/releases/tag/v0.17.9)